docs/application/web/guides/TOC.md

```diff
@@ -1,196 +1,197 @@
 # Application Management
 ## Applications
-### [Service Application](app-management/service-app-w.md)
-### [Watch Application](app-management/watch-app-w.md)
-### [Widget Application](app-management/web-widget-ww.md)
-## [Application Groups](app-management/app-group-w.md)
-## [Application Preferences](app-management/preference-w.md)
-## [Application Icons](app-management/app-icons-w.md)
+### [Service Application](app-management/service-app.md)
+### [Watch Application](app-management/watch-app.md)
```

Comment:
absent page : This watch-app.md page is left out by the author, so we don't have this page on git. It only exists on Tizen Developer Site. Thus, this link is broken.

Reply:
It was discussed with a maintainer. watch-app.md will be removed.